cmd/svcat/class/describe_cmd.go

```diff
@@ -76,12 +76,17 @@ func (c *describeCmd) Run() error {
 }
 
 func (c *describeCmd) describe() error {
-	var class *v1beta1.ClusterServiceClass
+	var class servicecatalog.Class
 	var err error
 	if c.lookupByUUID {
 		class, err = c.App.RetrieveClassByID(c.uuid)
 	} else {
-		retrieved, retErr := c.App.RetrieveClassByName(c.name, servicecatalog.ScopeOptions{Scope: servicecatalog.ClusterScope})
+		retrieved, retErr := c.App.RetrieveClassByName(c.name, servicecatalog.ScopeOptions{
+			Scope: servicecatalog.ClusterScope,
+		})
+		if retErr != nil {
+			return retErr
+		}
 		class = retrieved.(*v1beta1.ClusterServiceClass)
```

Comment:
Can we simplify this and drop the use of `retrieved` & `retErr` and instead use `class` and `err`?   I think you can remove lines 87-91 (I don't think the cast is necessary) and we end up with a simplified if/else statement.  WDYT?

Reply:
We get an error without the cast because all the retrieve methods currently only work with cluster-scoped resources. It should be fixed once #2215 is closed :)

Reply:
Oops, my bad! The problem was with the variable declaration. I fixed that. But the output method still accepts a `ClusterServiceClass`, so it'll still need a cast. I added a TODO to remove it later.

Reply:
Per #2215 this PR should change the function signature of `WriteClassDetails` to accept a class interface.

Reply:
Turns out that's already changed! Just had to remove the cast.